AppSettingsManagerAPI/AppSettingsManagerApi.Infrastructure/MySql/MySqlBaseUserRepository.cs

```diff
@@ -37,4 +37,33 @@ public async Task<Model.BaseUser> CreateUser(string userId, string password)
             _settingsContext.BaseUsers.Single(u => u.UserId == userId)
         );
     }
+    public async Task<Model.BaseUser> DeleteUser(string userId)
+    {
+        var user = await _settingsContext.BaseUsers.SingleOrDefaultAsync(u => u.UserId == userId);
```

Comment:
I would probably just use .Single() here rather than .SingleOrDefault()

.Single() will already throw an exception if the entity you specify isn't found so we don't have to check for null ourselves